pkg/csi_driver/controller.go

```diff
@@ -50,6 +51,8 @@ const (
 	enterpriseTierMaxSize = 10 * util.Tb
 	highScaleTierMinSize  = 10 * util.Tb
```

Comment:
Ideally we do not want to support both "high_scale_ssd" and "zonal" string. (i.e if user provides "high_scale_ssd" in the storage class we should deny the request). However let's double check with Filestore on what is the end of life for this stirng. The deprecation path for this would be tricky, where filestore would deprecate first and then GKE would rollout a change to not support the "high_scale_ssd" string in storageclass, and needs auto-upgrades etc. So we want to avoid all of that and just support the "zonal" string.
 
As long as static provisioning is not impacted by this, we should only support 1 tier string.

We need to map it to the definition of the bands https://cloud.google.com/filestore/docs/service-tiers#zonal_with_capacity_bands

Reply:
We have customers using `high_scale_ssd` now and we were not actively blocking it, blocking now would be a breaking change

Reply:
Yes, thats the concern. Does tier impact volume expansion or delete calls? 

Asking because, say starting GKE 1.x we support tier "zonal" only, it impacts only new PVC creation. Existing ones continue to work throughout its lifecycle.

Reply:
offline followed up, `high_scale_ssd` won't be deprecated.